perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py

```diff
@@ -7,8 +7,13 @@
 from typing import Any, Dict, List, Tuple
```

Comment:
to be clear, this is only used for the aria demo? do we use this class for the skills as well?

Reply:
currently object detector is only used in aria code. yes.